conf/application.conf

```diff
@@ -104,56 +113,67 @@ controllers {
 # You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
 
 # Root logger:
-logger.root=ERROR
+logger.root = ERROR
 
 # Logger used by the framework:
-logger.play=INFO
+logger.play = INFO
 
 # Logger provided to your application:
-logger.application=DEBUG
+logger.application = DEBUG
 
 # Metrics plugin settings - graphite reporting is configured on a per env basis
 metrics {
-    name = ${appName}
-    rateUnit = SECONDS
-    durationUnit = SECONDS
-    showSamples = true
-    jvm = true
-    enabled = true
+  name = ${appName}
+  rateUnit = SECONDS
+  durationUnit = SECONDS
+  showSamples = true
+  jvm = true
+  enabled = true
 }
 
 
 # Microservice specific config
 
 auditing {
-    enabled=true
-    traceRequests=true
-    consumer {
-        baseUri {
-            host = localhost
-            port = 8100
-        }
+  enabled = true
+  traceRequests = true
+  consumer {
+    baseUri {
+      host = localhost
+      port = 8100
     }
+  }
 }
 
 microservice {
-    metrics {
-        graphite {
-            host = graphite
-            port = 2003
-            prefix = play.${appName}.
-            enabled = false
-        }
+  metrics {
+    graphite {
+      host = graphite
+      port = 2003
+      prefix = play.${appName}.
+      enabled = false
```

Comment:
AFAIK we need legacy = false here.
This one was provided by reference.conf at bootstrap-play-25, but we don't have reference.conf anymore.